docs/wallets/fennec.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Fennec
+sidebar_position: 2
+---
+
+[Fennec](https://github.com/fox-one/fennec) is an open source browser extension that allows you to easily manage your own crypto assets wallet of Mixin Network.
+
+You can install Fennec from Chrome web store or Firefox Add-ons:
+
+- [Chrome Store](https://chrome.google.com/webstore/detail/fennec/eincngenkhohbbfpkohipekcmnkfamjp)
+- [Firefox Add-ons](https://addons.mozilla.org/en-US/firefox/addon/fennec/)
+
+Fennec uses Mixin Network's keystore files to identify your assets and manage your wallet. There are two ways to create your keystore files.
+
+By visiting [Mixin Developer Dashboard](https://developers.mixin.one/dashboard), creating an dApp and generating a keystore file.
+
+:::info
+**Mixin Developer Dashboard** is a service hosted by Mixin Core Team. You can use it to create two free dApps, but you will need to pay a fee for more dApps.
+:::
+
+By using the [Account Service providers](https://github.com/fox-one/fennec-asp), which generates a keystore file for you.
```

Comment:
The second option is to use the [Account Service providers](https://github.com/fox-one/fennec-asp) to generate a keystore file.